packages/core/src/guided-answers-api.ts

```diff
@@ -242,7 +244,7 @@ async function sendFeedbackComment(
     treeId: GuidedAnswerTreeId,
     nodeId: GuidedAnswerNodeId,
     comment: string
-): Promise<void> {
+): Promise<PostFeedbackResponse> {
     const message = comment;
     return await postFeedback(`${host}/${FEEDBACK_COMMENT}`, { treeId, nodeId, message });
```

Comment:
We can remove the await statement here
```suggestion
    return postFeedback(`${host}/${FEEDBACK_COMMENT}`, { treeId, nodeId, message });
```

Reply:
Done, thanks!